src/broker/log.c

```diff
@@ -231,11 +181,14 @@ void logbuf_destroy (logbuf_t *logbuf)
             logbuf_trim (logbuf, 0);
             zlist_destroy (&logbuf->buf);
         }
-        if (logbuf->sleepers) {
-            struct sleeper *s;
-            while ((s = zlist_pop (logbuf->sleepers)))
-                sleeper_destroy (s);
-            zlist_destroy (&logbuf->sleepers);
+        if (logbuf->followers) {
+            const flux_msg_t *msg;
+            while ((msg = zlist_pop (logbuf->followers))) {
+                if (flux_respond_error (logbuf->h, msg, ENODATA, NULL) < 0)
+                    log_err ("%s: flux_respond_error", __FUNCTION__);
+                flux_msg_decref (msg);
+            }
+            zlist_destroy (&logbuf->followers);
         }
```

Comment:
I missed this before, but here we might want to just destroy the "followers" and not send them a response.  `logbuf_destroy()` is called as an aux destructor on the broker's `flux_t`.  This would occur after all broker TBON children have exited, and after the local connector has been unloaded.  Therefore, there's really no place for the response to go.